### PR TITLE
ETS-1308 adds style for QS w background

### PR DESCRIPTION
**Pull request by virgilchiriac**

### CODE REVIEW

File: source/scss/components/_quicksearch.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.qs-with-background {
+  bottom: 150px;
+  margin-bottom: -100px;
+  position: relative;
+  @media(max-width: 1279px) {
```
Comment (cllagosr) [inline source/scss/components/_quicksearch.scss]: you should use mixins for all the breakpoints

Comment (virgilchiriac) [reply source/scss/components/_quicksearch.scss]: was fixed

**Review by cllagosr — DISMISSED** [review]
update changelog please.

File: source/_patterns/02-organisms/07-quicksearch/00-quicksearch-image.mustache [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<div class="page-headline">
+  <div class="container">
+    <h1 class="qs-h1">{{ pageHeadline }}</h1>
+  </div>
+</div>
+
+<div class="qs-background">
+  <picture>
```
Comment (BirgitPohl) [inline source/_patterns/02-organisms/07-quicksearch/00-quicksearch-image.mustache:8]: picture-tag is not supported in ie11
https://caniuse.com/#search=picture

Do you have a fallback?

File: source/scss/components/_quicksearch.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    margin-bottom: -90px;
+  }
+  @include media-breakpoint-between("md", "lg") {
+    bottom: 250px;
+    margin-bottom: -200px;
+  }
+}
+
+.qs-h1 {
+  font-size: 3em;
```
Comment (BirgitPohl) [inline source/scss/components/_quicksearch.scss]: Do you want to add the typography values to the custom-variables?
Maybe you find some useful values there.
Also em does not use the html font-size, but the parent's font-size. Is this intended?

File: source/_patterns/02-organisms/07-quicksearch/00-quicksearch-image.mustache (hunk 2 above)
Comment (virgilchiriac) [reply source/_patterns/02-organisms/07-quicksearch/00-quicksearch-image.mustache:8]: there is a img tag as well. In any case, same thing we use for teasers

**Review by cllagosr — DISMISSED** [review]

**Review by cllagosr — APPROVED** [review]

**Pull request merged**